### ISSUE TITLE
Polymer core-animated-pages crash during transition resizing

### ISSUE THREAD

**Issue by kiempoturner** [Polymer/core-animated-pages#45]
[http://stackoverflow.com/questions/28411829/polymer-core-animated-pages-crash-during-transition-resizing]

As the title says, I have 5 pages, the first 3's heights are scrollable while the last 2 pages are at their fixed height.

The last 2 pages will crash whenever I selected the tab they're on. One thing I notice is that if I set both pages' height to a fair amount of scrollable height, it will not crash, but that wont solve the problem as I need to make it fixed.

I already tried setting them to ````position:fixed;```` & ````overflow-y:hidden;```` . The same thing will occur. Crashed.

Obviously the reason behind this is because the resizing between the transitions.

Does anyone has any idea why? and how to fix it? Really need your help.

*ps: crash on cordova running on android 4.4*

**Comment by MukeshDangodara** [comment]
Hi,
I am working on Polymer animated pages. I want to add core-button-icons . I get an error while add button like  "Polymer is not a function".
can anyone help ?

**Comment by kiempoturner** [comment]
````core-icon-button```` instead of ````core-button-icons```` perhaps?

**Comment by MukeshDangodara** [comment]
Yeah its core-icon-buttons. its not issue. I think js is conflict or
something else is creating issue.
Can you suggest me some tutorial for best polymer example ?

**Comment by kiempoturner** [comment]
unrecognizable elements will always return errors. 

take a look at their components and understand it.

https://www.polymer-project.org/components/core-elements/demo.html#core-animation
https://www.polymer-project.org/components/paper-elements/demo.html#paper-checkbox

for learning, I'd suggest you to learn by existing templates. check out ````yo polymer````

**Comment by MukeshDangodara** [comment]
ok,thank you.
